.github/ISSUE_TEMPLATE/bug-issue.md

```diff
@@ -0,0 +1,47 @@
+---
+name: Bug Issue
+about: Use this template for reporting a bug
+labels: 'bug'
+title: '[BUG]'
+
+---
+
+## Bug
+
+### Environment information
+
+* Delta Lake version:
+* Spark version:
+* Scala version:
+
+### Describe the problem
+
+#### Steps to reproduce
+
+<!--
```

Comment:
Could we add a note like "please include copy-pastable code snippets if possible."

Reply:
Ok.